files/logstash-configs/22-redmine.conf

```diff
@@ -23,6 +23,9 @@ filter {
         match => {"message" => "%{REDMINE_SUCCESSFUL_LOGIN}"}
         add_tag => "successful_login"
       }
+      date {
+        match => ["redmine_auth_timestamp", "yyyy-MM-dd HH:mm:ss"]
```

Comment:
Any reason not to fold `redmine_auth_timestamp` into `redmine_timestamp`? The patterns are different, but both values should constitute the event timestamp in the elasticsearch.